en/clubs/internationellanamnden/body.md

```diff
@@ -1,9 +1,29 @@
-# The International Club
+# International Committee (Internationella nämnden)
 
-The International Club is the Computer Science Chapter's International Group!
+The International Committee of the Computer Science Chapter is responsible for all international activities of the Computer Science Chapeter and is shared with the Media Technology Chapter. Led by the Data Chapter's International Student Coordinator (Valentin Quapil) and the Chapter for Media Technology's International Coordinator (Lovisa Condé).
 
-This is a shared group between the Computer Science Chapter and the Chapter for Media Technology. The group organizes at least two events per semester for computer science students and media technology students. The group recruits new excited organizers at the beginning of each semester, but is open to new organizers all year round.
 
-By helping out in the group organizers can collect hours to count toward their 0.2 GPA bonus to the GPA points that are used to apply for international studies. The group leaders are responsible for registering the hours with THS. If you are interested in joining the group as an organizer, then apply for membership in the organizers' group: [Facebook group](https://www.facebook.com/groups/2151534464869750/)
+## General
 
-Also, feel free to like the group's facebook page in order to receive news of the group's activities: [Facebook page](https://www.facebook.com/METAInternationalGroup/)
+The International Committee focuses on introducing international students to our Chapter. This includes organising events for international students, helping international students to participate in Computer Science Chapter activities and providing a platform for cultural exchange between Swedish and international students. The committee is open to all students in the Computer Science Chapter, regardless of whether you are international or not.
+
+Please like the committee's Facebook page to receive updates on the group's events: [Facebook page](https://www.facebook.com/METAInternationalGroup/?eid=ARDdqkLUe2uRu9PZyhqSOZSHFii_V4D0Pyur3l29_TbwBwnJdzM-PPi6HzHEvk_GjVNlp7DBRUWzpQBj)
+
+# Join the committee
+
+By getting involved in the committee, organizers can earn hours towards the 0.2 bonus points used to apply for study abroad. It takes 25 hours of involvement over 4 months to earn the bonus points, and the Head of the Committee (International Student Coordinator) is responsible for counting the hours and recording them with THS.
```

Comment:
Nit: there are some inconsistencies with the spelling of organize/organise. Use either s or z ;)

Is recording the best word? Would probably use report.

Reply:
I dont know if there is some standard way inside of Dsekt, I decided for using z for now.